kura/examples/org.eclipse.kura.example.beacon/OSGI-INF/metatype/org.eclipse.kura.example.beacon.BeaconExample.xml

```diff
@@ -81,6 +87,8 @@
             cardinality="0"
             required="true"
             default="0"
+            min="0"
+            max="255"
             description="Transmission power measured at 1m away from the beacon expressed in dBm."/>
```

Comment:
I'm not sure that the limits for TxPower should be 0 to 255. I think that -127 to +126 is better since it is expressed in dBm.